src/contexts/MintBurnFormContext/MintBurnFormContext.tsx

```diff
@@ -267,6 +268,26 @@ export const MintBurnFormProvider: React.FunctionComponent<MintBurnFormProviderP
     }
   }, [fixedHigh, fixedLow, liquidityAction, margin, marginAction, notional, minimumRequiredMargin]);
 
+  const notionalChange = useCallback(debounce((value: number | undefined) => {
+    pushEvent("notional_change", value ?? 0, sessionId, poolAmm, "Liquidity Provider");
+  }, 1000), []);
+
+  const marginChange = useCallback(debounce((value: number | undefined) => {
+    pushEvent("margin_change", value ?? 0, sessionId, poolAmm, "Liquidity Provider");
+  }, 1000),[]);
+
+  useEffect(() => {
+    if ( !isUndefined(mintMinimumMarginRequirement.result)) {
```

Comment:
can you add a task in the FE UI where we can refactor the isUndefined to support null. I feel that we might be inviting bugs if someone passes null due to the component props allowing it.

Reply:
I'm adding this to a new ticket bc it needs a lot of refactoring that's out of scope for this PR